bin/ajax

```diff
@@ -335,6 +356,8 @@ def clean_abandoned(delete_live=False):
     # runs to be deleted simultaneously.
     if not delete_live:
         rd = run_coll.find_one({'bootstrax.state': 'abandoned',
+                                'bootstrax.time':
+                                    {"$lt": now(-ajax_thresholds['wait_after_processing'])},

```

Comment:
Is this intended to protect us against accidental abandonment? I'm not sure if this is particularly likely to happen.